From 2594e9458b82d305a0bccf5ee77b0c6c189639c7 Mon Sep 17 00:00:00 2001
From: Noam Camus <ncamus@habana.ai>
Date: Fri, 24 Aug 2018 12:08:09 +0300
Subject: [PATCH] Makefile [hl-distclean] add new goal

It is now using HL_DEV to decide which distclean to use
---

diff --git a/Makefile b/Makefile
index 29abe5a..239db9d 100644
--- a/Makefile
+++ b/Makefile
@@ -193,6 +193,12 @@
 		thermal-custom preboot-custom host-fw_loader-custom; \
 	fi
 
+.PHONY: hl-distclean
+hl-distclean:
+ifneq ($(HL_DEV),)
+	@$(MAKE) $(HL_DEV)-distclean
+endif
+
 .PHONY: dali-onix-test dali-pldm-test dali-asic-test
 dali-onix-test: dali-onix-tools-install
 dali-onix-test dali-pldm-test dali-asic-test:
